recipes/libcurl/all/conanfile.py

```diff
@@ -584,6 +592,10 @@ def _generate_with_cmake(self):
             tc.variables["CURL_USE_WOLFSSL"] = self.options.with_ssl == "wolfssl"
         else:
             tc.variables["CMAKE_USE_WOLFSSL"] = self.options.with_ssl == "wolfssl"
+        if Version(self.version) >= "7.81.0":
+            tc.variables["CIRL_USE_MBEDTLS"] = self.options.with_ssl == "mbedtls"
```

Comment:
typo?

```suggestion
            tc.variables["CURL_USE_MBEDTLS"] = self.options.with_ssl == "mbedtls"
```

Reply:
thanks for review, it' s a mistake, shoud I restart a new pr?

Reply:
No, you can just update this one and CI should automatically retrigger. Thanks!